connect/src/com/telenor/connect/AbstractSdkProfile.java

```diff
@@ -92,4 +92,11 @@ public void failure(RetrofitError error) {
                     }
                 });
     }
+
+    protected String applyStagingOnEndpoint(String endpoint) {
```

Comment:
I don't think this has to be a member of the base class.
It's kind of a hack, so we'd better have it as limited as possible.
Plus, in the future (when you are ready with the tests), you want that removed for the MC profile, and then it make no sense to have it here. This is purely ConnectId specific thing.